src/resources/users/repositories/MongoDBUsersRepository.ts

```diff
@@ -1,18 +1,33 @@
 import { Inject, Service } from "typedi";
 import { Pagination } from "../../../common/parameters/Pagination";
 import { MongoDBConnector } from "../../../common/services/MongoDBConnector";
-import { MONGO_DB_USER_DEFAULT_PROJECTION } from "../../../config/Config";
+import { MONGO_DB_DEFAULT_PROJECTION, MONGO_DB_USER_DEFAULT_PROJECTION } from "../../../config/Config";
 import { CreateUserRequest } from "../../../generated/models/CreateUserRequest.generated";
 import { UpdateUserRequest } from "../../../generated/models/UpdateUserRequest.generated";
 import { User } from "../../../generated/models/User.generated";
 import { generateID } from "../../../utils/IDUtil";
 import { createMetadata, getUpdatedMetadata } from "../../../utils/MetadataUtil";
 import { UsersRepository } from "./UsersRepository";
+import { Membership } from "../../../generated/models/Membership.generated";
+import { Filter } from "../../../generated/models/Filter.generated";
+import { UpdateOrganizationMembershipRequest } from "../../../generated/models/UpdateOrganizationMembershipRequest.generated";
 
 @Service()
 export class MongoDBUsersRepository implements UsersRepository {
 	constructor(@Inject("DBClient") private dbConnector: MongoDBConnector) {}
 
+	async get(filter?: Filter, projection?: any, pagination?: Pagination): Promise<any[]> {
```

Comment:
Hier können wir `User` statt `any` für den Rückgabewert definieren, oder?

```suggestion
	async get(filter?: Filter, projection?: any, pagination?: Pagination): Promise<User[]> {
```

Reply:
jut